docs/source/notebooks/lda-advi-aevb.ipynb

```diff
@@ -718,7 +718,9 @@
     }
```

Comment:
<p>the trailing backslashes are necessary to retain validity of the in-line magic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>